js-miniapp-bridge/src/types/message-to-contact.ts

```diff
@@ -8,4 +8,6 @@ export interface MessageToContact {
   caption: string;
   // Action which the call-to-action button will perform, i.e. a URI with parameters.
   action: string;
+  // Optional 128 characters message which can be used on contacts picker list for example
+  bannerMessage: string;
```

Comment:
Including a message is optional, so this should be:
```suggestion
  bannerMessage?: string;
```